pylint/config/option_manager_mixin.py

```diff
@@ -236,22 +236,24 @@ def load_provider_defaults(self):
             provider.load_defaults()
 
     def read_config_file(self, config_file=None, verbose=None):
-        """read the configuration file but do not load it (i.e. dispatching
-        values to each options provider)
-        """
-        helplevel = 1
-        while helplevel <= self._maxlevel:
-            opt = "-".join(["long"] * helplevel) + "-help"
+        """Read the configuration file but do not load it (i.e. dispatching
+        values to each options provider)"""
```

Comment:
```suggestion
        """Read the configuration file but do not load it (i.e. dispatching
        values to each options provider)
        """
```
For multiline comments it's usually common practice to put the closing quotes on a new line.

Reply:
Just FWI, this wasn't the only related change with this issue :)

Reply:
I think PcManticore prefer the other way to do it that's why I do it like this. I actually like the version that take less lines too. I wish black would automatically do that. Also I think we should let those the way they are and not bother about it.

Reply:
I would imaging black would choose the PEP 8 [style](https://www.python.org/dev/peps/pep-0008/#documentation-strings) though, but I agree it's not worth arguing about small things like that.

Reply:
Ho if it's pep8 then I'm going to do it the other way next time, thanks for pointing that out.